src/core/interfaces.ts

```diff
@@ -1,11 +1,13 @@
+import { ILoggerProps } from '../logging/logging';
+
 export interface IConfig {
   root?: string;
   target?: 'browser' | 'server' | 'electron' | 'universal';
   homeDir?: string;
   output?: string;
   modules?: Array<string>;
   fuseBoxPolyfillsFolder?: string;
-
+  logging?: ILoggerProps;
```

Comment:
Do you think we could add `boolean` there? So user could just disable all the logs ?